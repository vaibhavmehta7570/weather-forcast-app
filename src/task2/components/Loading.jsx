import React from 'react'
import '../loading.css'

const Loading = () => {
  return (
    <div class="loadingio-spinner-spinner-rwrfifmnsar">
      <div class="ldio-uvsg4dk6pgq">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>
  )
}

export default Loading
