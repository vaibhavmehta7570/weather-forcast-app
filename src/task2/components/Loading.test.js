import React from 'react'
import Loading from './Loading'
import renderer from 'react-test-renderer'

describe('should test app.js', () => {
  test('should render Loading div correctly', () => {
    const appRenderResponse = renderer.create(<Loading />).toJSON()
    expect(appRenderResponse).toMatchSnapshot()
  })
})
