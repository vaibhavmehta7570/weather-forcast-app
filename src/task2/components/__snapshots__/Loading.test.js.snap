// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test app.js should render Loading div correctly 1`] = `
<div
  class="loadingio-spinner-spinner-rwrfifmnsar"
>
  <div
    class="ldio-uvsg4dk6pgq"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
